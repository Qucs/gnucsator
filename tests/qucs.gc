load lang_qucs.so
* this is a hack.
* we don't have a --language switch...
qucs
