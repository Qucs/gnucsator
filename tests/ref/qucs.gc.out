;: already installed, replacing
stashing as ;:0
*: already installed, replacing
stashing as *:0
': already installed, replacing
stashing as ':0
": already installed, replacing
stashing as ":0
dev_comment: already installed, replacing
stashing as dev_comment:0

